Projects/Dotmim.Sync.PostgreSql/Dotmim.Sync.PostgreSql.csproj

```diff
@@ -1,14 +1,55 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFramework>netstandard2.0</TargetFramework>
+		<TargetFrameworkNetStandard>netstandard2.0</TargetFrameworkNetStandard>
+		<TargetFrameworkNet6>net6.0</TargetFrameworkNet6>
+		<TargetFrameworkNet7>net7.0</TargetFrameworkNet7>
+		<TargetFrameworkNetCore>netcoreapp3.1</TargetFrameworkNetCore>
+
+		<TargetFrameworks>$(TargetFrameworkNetStandard);$(TargetFrameworkNet6);$(TargetFrameworkNet7);$(TargetFrameworkNetCore)</TargetFrameworks>
+
+		<Authors>Sébastien Pertus</Authors>
+		<Company>Microsoft</Company>
+		<Title>Dotmim.Sync.MySql</Title>
+		<Summary>MySql Sync Provider. Client or Server provider .Net Standard 2.0</Summary>
+		<Description>MySql Sync Provider. Manage a sync process beetween two relational databases provider. This provider works with SQL Server and can be used as Client or Server provider .Net Standard 2.0</Description>
+		<PackageTags>Data, Database, Sync, Synchronization, Framework, Sql, MySql, Sqlite, Dotmim.Sync, NetStandard, SyncFramework</PackageTags>
+		<PackageProjectUrl>https://dotmimsync.readthedocs.io/</PackageProjectUrl>
+		<RepositoryUrl>https://github.com/Mimetis/Dotmim.Sync</RepositoryUrl>
+		<RepositoryType>git</RepositoryType>
+		<VersionSuffix>$(VersionSuffix)</VersionSuffix>
+		<Version>0.9.8</Version>
+		<Version Condition=" '$(VersionSuffix)' != '' ">$(Version)-$(VersionSuffix)</Version>
 		<LangVersion>9.0</LangVersion>
+		<ApplicationIcon>favicon.ico</ApplicationIcon>
+		<PackageIcon>packageIcon.png</PackageIcon>
+		<PublishRepositoryUrl>true</PublishRepositoryUrl>
+		<EmbedUntrackedSources>true</EmbedUntrackedSources>
+		<GenerateDocumentationFile>False</GenerateDocumentationFile>
+
 	</PropertyGroup>
 
+
 	<ItemGroup>
+		<ProjectReference Include="..\Dotmim.Sync.Core\Dotmim.Sync.Core.csproj" />
+		<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
+	</ItemGroup>
+
+	<ItemGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNet7)'">
+		<PackageReference Include="Npgsql" Version="7.0.1" />
+	</ItemGroup>
+	
+	<ItemGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNet6)'">
```

Comment:
we need to consider the breaking changes in Npgsql library from 5 to 6 and 6 to 7. specially with datetime

https://www.npgsql.org/doc/release-notes/6.0.html#major-changes-to-timestamp-mapping